### ISSUE TITLE
Why % css doesn't work 

### ISSUE THREAD

**Issue by antoninlanglade** [michaelvillar/dynamics.js#34]
I try to use css percent value and the result is bad. The animate func try to animate to the value, if you put 50% on a width for example, it will translate to 50px and in the end of animation set 50%. Strange result

**Comment by michaelvillar** [comment]
Yeah, the way that dynamics works, it's difficult (to not say impossible) to make it work for percentage units. Because dynamics only ask for the end value, dynamics needs to retrieve the current value from the DOM, and that current value is in absolute units (pixels), not relative (percentage).

**Comment by antoninlanglade** [comment]
Thanks for your reply. So bad ... In GSAP TweenMax we can. It's an important feature because when you are handling with DOM values like width, translate ... % it's very useful.
You have some cool stuff that GSAP don't have so in the future I will use the two libraries.
